### PR TITLE
ipset: Add InstallDev to provide libipset as library

### PR DESCRIPTION
**Pull request by corny**
I use `libipset` in another package.

### CODE REVIEW

File: package/network/utils/ipset/Makefile [hunk 1]
```diff
@@ -1,4 +1,4 @@
- 
+
 # Copyright (C) 2009-2012 OpenWrt.org
```
Comment (blogic) [inline package/network/utils/ipset/Makefile]: please just delete the whole line

**Comment by blogic** [comment]
the patch is missing the Signed-off-by line. please add it so that we can merge the patch

**Comment by corny** [comment]
done

**Comment by blogic** [comment]
merged, Thanks !

**Comment by corny** [comment]
Thanks for merging!

**Pull request closed without merging**